packages/cli/src/tools/packageManager.js

```diff
@@ -11,6 +11,32 @@ type Options = {|
 
 let projectDir;
 
+const yarnConfig = {
+  dependencies: ['add'],
+  devDependencies: ['add', '-D'],
+  uninstallDependencies: ['remove'],
+};
+
+const npmConfig = {
+  dependencies: ['install', '--save', '--save-exact'],
+  devDependencies: ['install', '--save-dev', '--save-exact'],
+  uninstallDependencies: ['uninstall', '--save'],
+};
+
+function configurePackageManager(
+  pmIsYarn: boolean,
+  packageNames: Array<string>,
+  options?: Options,
+  installType: string,
+) {
+  const pm = pmIsYarn ? 'yarn' : 'npm';
+  const pmConfig = pm === 'npm' ? npmConfig : yarnConfig;
+
+  let args = pmConfig[installType];
```

Comment:
What might be a better approach to conditionally assign `args`?

Reply:
`packageNames` seem to come up in the second place within the `args` array.

Reply:
You could do some thing like this:

```js
const [executable, flags] = pmConfig[installType];
const args = [executable, ...packageNames, ...flags];
```